packages/connect-known-chains/scripts/build.js

```diff
@@ -0,0 +1,40 @@
+// @ts-check
+import path from "path"
+import { fileURLToPath } from "url"
+import { readdir } from "node:fs/promises"
+import { spawn } from "child_process"
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+
+const jsDir = path.join(__dirname, "../src/specs/")
+
+const files = await readdir(jsDir)
+const tsFiles = files.filter((file) => file.endsWith(".ts"))
+
+const paths = tsFiles.map((fileName) => `src/specs/${fileName}`)
+
+const child = spawn("tsup-node", [
```

Comment:
If there is an error in this subprocess, the parent process exit code should be the subprocess exit code and not `0`